sample/build.gradle

```diff
@@ -0,0 +1,31 @@
+apply plugin: 'com.android.application'
+
+// Read API Key from gradle.properties
+def TURN_BY_TURN_API_KEY = hasProperty('valhallaApiKey') ? '"' + valhallaApiKey + '"' : "null";
+
+android {
+  compileSdkVersion 23
+  buildToolsVersion "23.0.3"
+
+  defaultConfig {
+    applicationId "com.mapzen.sample"
+    minSdkVersion 15
+    targetSdkVersion 23
+    versionCode 1
+    versionName "1.0"
+
+    // Create BuildConfig field for accessing gradle property in code
+    buildConfigField "String", "TURN_BY_TURN_API_KEY", TURN_BY_TURN_API_KEY
+  }
+  buildTypes {
+    release {
+      minifyEnabled false
+      proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
```

Comment:
Since we're not using proguard we probably want to take this out. Also all `proguard-rules.pro` files can be removed.